integration-tests/src/bin/gen_blockchain_data.rs

```diff
@@ -78,8 +78,16 @@ async fn main() {
     let mut contracts = HashMap::new();
     for (name, contract_path) in CONTRACTS {
         let path_sol = Path::new(CONTRACTS_PATH).join(contract_path);
+        let inputs = CompilerInput::new(&path_sol).expect("Compile success");
+        let input = inputs
+            .clone()
+            .first_mut()
+            .expect("first exists")
+            .clone()
+            .evm_version(EvmVersion::London);
```

Comment:
Could you have some comments to explain why we don't need to set evm version before but we need it now? Or we just have to set evm version here bcs it's part of rules of 2.0.7? Just would like to know what kind of situations we might need to upgrade the version in the future.

Reply:
Good point that we should add comments here. I added the comment in [e118683](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1452/commits/e1186830fd1571c43a227a11548d0fd2c4dd1a07)

I checked ethers.rs and found it defaulted [evm version to Shanghai](https://github.com/gakonst/ethers-rs/blob/7b7c62327303866319a8b9ea84de517eef70dc09/ethers-solc/src/artifacts/mod.rs#L756), which is incompatible with my local `0.8.16` version.


I printed out the CI solidity [version](https://github.com/privacy-scaling-explorations/zkevm-circuits/actions/runs/5473937281/jobs/9968062166?pr=1452#step:10:96), which is `0.8.19`. The Solidity [doc](https://docs.soliditylang.org/en/v0.8.19/using-the-compiler.html) for `0.8.19` says Paris is the latest supported EVM version. So if the ethers-solc selects Shanghai as EVM version, the Solidity compiler would not recognize it.